src/common/hooks/usePiecesMutation/usePiecesMutation.js

```diff
@@ -0,0 +1,20 @@
+import { useMutation } from 'react-query';
+
+import { ORDER_PIECES_API } from '@folio/stripes-acq-components';
+import { useOkapiKy } from '@folio/stripes/core';
+
+const usePiecesMutation = (options = {}) => {
+  const ky = useOkapiKy();
+
+  const { mutateAsync, isLoading } = useMutation({
+    mutationFn: (piece) => ky.put(`${ORDER_PIECES_API}/${piece.id}`, { json: piece }),
+    ...options,
+  });
+
+  return {
+    isLoading,
+    unboundPiece: mutateAsync,
```

Comment:
too specific function name for a hook called `usePieceMutation`: this fn updates the piece, not just "unbound"